face-stylizer/index.ts

```diff
@@ -0,0 +1,173 @@
+// Copyright 2022 The MediaPipe Authors.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//      http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { FaceStylizer, FilesetResolver } from '@mediapipe/tasks-vision';
+
+const demosSection = document.getElementById("demos");
+
+let faceStylizer: FaceStylizer;
+let runningMode: "IMAGE" | "VIDEO" = "IMAGE";
+let enableWebcamButton: HTMLButtonElement;
+let webcamRunning: Boolean = false;
+const videoWidth = 480;
+
+// Before we can use FaceStylizer class we must wait for it to finish
+// loading. Machine Learning models can be large and take a moment to
+// get everything needed to run.
+async function runDemo() {
+  // Read more `CopyWebpackPlugin`, copy wasm set from "https://cdn.skypack.dev/node_modules" to `/wasm`
+  const vision = await FilesetResolver.forVisionTasks("/wasm");
+  faceStylizer = await FaceStylizer.createFromOptions(vision, {
+    baseOptions: {
+      modelAssetPath: `https://storage.googleapis.com/mediapipe-assets/face_stylizer_cartoon.task`
+    }
+  });
+  demosSection.classList.remove("invisible");
+}
+runDemo();
+
+/********************************************************************
+// Demo 1: Grab a bunch of images from the page and detection them
+// upon click.
+********************************************************************/
+
+// In this demo, we have put all our clickable images in divs with the
+// CSS class 'detectionOnClick'. Lets get all the elements that have
+// this class.
+const imageContainers = document.getElementsByClassName("detectOnClick");
```

Comment:
const imageContainers = document.getElementsByClassName(
  "detectOnClick"
) as HTMLDivElement;

for (let imageContainer of imageContainers) {
  imageContainer.children[0].addEventListener("click", handleClick);
}

Reply:
The result of `document.getElementsByClassName("detectOnClick")` is a `HTMLCollectionOf<Element>`.
So that, we can not using `for ... let`, because it's not an array.

I change `as HTMLDivElement` to `as HTMLCollectionOf<HTMLDivElement>`